lua/cmp/config.lua

```diff
@@ -26,7 +27,7 @@ config.onetime = {}
 ---Set configuration for global.
 ---@param c cmp.ConfigSchema
 config.set_global = function(c)
-  config.global = misc.merge(config.normalize(c), config.normalize(config.global))
```

Comment:
Is this okay? I'm not sure if you normalized before merging to avoid a specific bug.

See the commit for details on why I did it in this order; it has to do with detecting the user config in time to normalize it because now with `bordered`/`default`, default config values depend on the user config.